perf/bloom_filter/CMakeLists.txt

```diff
@@ -0,0 +1,9 @@
+set(CMAKE_CXX_STANDARD 11)
+
+add_library(bench SHARED bloom_filter.cc)
+# for xmalloc
+target_link_libraries(bench unit)
```

Comment:
Why do you link with `unit`?

Reply:
Because I thought xmalloc is implemented in `unit`. Isn't it true?

Reply:
Deleted this line. Compiling without it.